quidel_covidtest/delphi_quidel_covidtest/run.py

```diff
@@ -60,15 +61,12 @@ def generate_and_export_for_nonparent_geo(geo_groups, res_key, smooth, device,
                                           first_date, last_date, suffix, # generate args
                                           geo_res, sensor_name, export_dir,
                                           export_start_date, export_end_date, # export args
-                                          lock, log_filename, log_exceptions): # logger args
+                                          threaded_logger): # logger args
     """Generate sensors, create export CSV then return stats."""
-    # logger cannot be passed to child processes, so has to be recreated
-    with lock:
-        logger = get_structured_logger(__name__, log_filename, log_exceptions)
-        logger.info("Generating signal and exporting to CSV",
-                geo_res=geo_res,
-                sensor=sensor_name,
-                pid=current_process().pid)
+    threaded_logger.info("Generating signal and exporting to CSV",
+                         geo_res=geo_res,
+                         sensor=sensor_name,
+                         pid=current_process().pid)
```

Comment:
not needed!
```suggestion
                         sensor=sensor_name)
```
it already does this for you: https://github.com/cmu-delphi/covidcast-indicators/blob/70e0263040df6adaf5c6656032d540f859d545ad/_delphi_utils_python/delphi_utils/logger.py#L159
(same for other `generate_...()` method below this)